src/edu/washington/escience/myria/operator/network/partition/HowPartitioned.java

```diff
@@ -0,0 +1,60 @@
+package edu.washington.escience.myria.operator.network.partition;
+
+import java.util.Set;
+
+import javax.annotation.Nullable;
+
+import jersey.repackaged.com.google.common.collect.ImmutableSet;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.primitives.Ints;
+
+/**
+ * How a dataset is partitioned. There are two fields being recorded for now, the partition function and the sequence of
+ * worker IDs. More information can be added in the future if needed.
+ */
+public class HowPartitioned {
+
+  /** The partition function which was used to partition the dataset. Null means unknown. */
+  @JsonProperty
+  private PartitionFunction pf = null;
+  /** The sequence of workers that the dataset is partitioned on. Null means unknown. */
+  @JsonProperty
+  private int[] workers = null;
```

Comment:
I guess List is the same as an int[] regarding JSON mapping..? I didn't use Set because JSON doesn't have Set as a data type and I simply want to avoid confusion.